public/app/features/browse-dashboards/api/browseDashboardsAPI.ts

```diff
@@ -125,14 +123,12 @@ export const browseDashboardsAPI = createApi({
       onQueryStarted: ({ parentUid }, { queryFulfilled, dispatch }) => {
         queryFulfilled.then(async ({ data: folder }) => {
           await contextSrv.fetchUserPermissions();
-          dispatch(notifyApp(createSuccessNotification('Folder created')));
```

Comment:
Moved to the consumer component (`browse-dashboards/components/CreateNewButton`)